tasties/urls.py

```diff
@@ -20,5 +20,6 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', views.index, name="index"),
-    path('base/', views.base, name="base")
+    path('base/', views.base, name="base"),
+    path('recipes/', views.recipes, name="recipes")
```

Comment:
Maybe it could be nice to use the `/` path for the recipes instead of `index` which was something temporary

Reply:
I haven't done this change yet because I want the authentication to merged, so we could add `@login_required` to the recipes template and to be redirected to the login page if user is not atuhenticated.

Reply:
Thanks for the clarification